src/services/github.ts

```diff
@@ -39,7 +38,19 @@ export async function publishNewDao(
       token_logo: daoMetadata.token_logo
         ? `${ASSETS_FOLDER_BASE}/${daoMetadata.name}/token_logo.${daoMetadata.token_logo.imageExtension}`
         : null,
-    })
+    }
+
+    newDaoList.push(
+      daoMetadata.wrappableToken === null
```

Comment:
Here and in the line 295 we have 2 differnts outcomes.

Check that tests about undefined and null 

![image](https://user-images.githubusercontent.com/691510/150399691-5e2c6395-ce3e-4a73-99f0-19be63c84e63.png)

On GithubMetadata we will set null to wrappableToken or undefined?

The checks are differents.

I'll suggest is keep the pattern, setting null so we need change the types, removing the optional and putting | null on wrappableToken

![image](https://user-images.githubusercontent.com/691510/150400038-fc0b3c48-7e40-4625-88ee-56b13a301903.png)


And add the default null - line 32
![image](https://user-images.githubusercontent.com/691510/150400549-15beec26-5fd2-4b82-bab7-f368849e2258.png)

I could commit that changes if needed.

Reply:
Accordingly to Gabi, the wrappableToken should not be sent